signbank/dictionary/tasks.py

```diff
@@ -66,6 +65,8 @@ def retrieve_videos_for_glosses(video_details: List[VideoDetail]):
     videos_to_create = []
 
     temp_dir = TemporaryDirectory(dir=settings.MEDIA_ROOT)
+    if not settings.DEBUG and settings.GLOSS_VIDEO_FILE_STORAGE == "storages.backends.s3boto3.S3Boto3Storage":
+        temp_dir = TemporaryDirectory(dir=settings.PROJECT_DIR)
```

Comment:
**question**: does this not work with `temp_dir = TemporaryDirectory()`?
If no parent directory is specified, the operating system should give the user a directory that they can write to

Reply:
This is now guesswork. It should work with `temp_dir = TemporaryDirectory()`, it seems like it only locally needs to be the media root.